app/controllers/editor_controller.rb

```diff
@@ -5,6 +5,7 @@ class EditorController < ApplicationController
   def legacy
     # /post/?i=http://myurl.com/image.jpg
     flash.now[:notice] = "This is the legacy editor. For the new rich editor, <a href='/editor'>click here</a>."
+    flash.now[:error] = "Deprecation notice: Legacy editor will be discontinued soon, please use rich/markdown editor."
```

Comment:
```suggestion
    flash.now[:warning] = "Deprecation notice: Legacy editor will be discontinued soon, please use rich/markdown editor."
```

How about using warning alert as error seems a bit more? What do you think @CleverFool77? Thanks!

Reply:
Hi @gauravano 
I've updated The PR as requested. Thanks !!!